lib/vcloud/launcher/launch.rb

```diff
@@ -5,11 +5,11 @@ class Launch
       class MissingPreambleError < RuntimeError ; end
       class MissingConfigurationError < RuntimeError ; end
 
-      attr_reader :config, :cli_options
+      attr_reader :config, :options
 
-      def initialize(config_file, cli_options = {})
```

Comment:
Just one point to raise here is that it seems a little ugly to have config_file as the first option, then all the other options as a hash. I'd rather just pass a hash in, I was focusing on kill Cli, but happy to clean this up if it makes sense to do so.